data/dataverses/manchester/datasets/test-dataset/test-dataset.json

```diff
@@ -28,7 +28,7 @@
             "typeName": "title",
             "multiple": false,
             "typeClass": "primitive",
-            "value": "Test Dataset"
+            "value": "Test Dataset with Shapefiles, Images, and Tabular Files"
```

Comment:
I would consider removing the word "test" entirely.

Reply:
Done